.github/workflows/ci.yml

```diff
@@ -34,10 +34,10 @@ jobs:
         go test ./...
     # We set the maximum version of the go directive as 1.20 here
     # because the oldest go directive that exists on our supported
-    # release branches in k/k is 1.20.
+    # release branches in k/k is 1.22.8.
```

Comment:
Oldest go directive in 1.29 is still go 1.21: https://github.com/kubernetes/kubernetes/blob/release-1.29/go.mod#L9